press/utils.py

```diff
@@ -2,9 +2,20 @@
     'convert_to_legacy_domain',
     'convert_version_tuple_to_version_string',
     'convert_version_to_legacy_version',
+    'requires_major_version_update',
 )
 
 
+def requires_major_version_update(before, after):
```

Comment:
Bit of scope creep, but a good addition none the less. Though, I'd have liked to see this as a stand alone PR.

Reply:
I see! Agreed; although, it came in very handy as a means to test that the rest of my code was working properly, for this same PR.